keypoints/include/pcl/keypoints/impl/iss_3d.hpp

```diff
@@ -152,7 +152,7 @@ pcl::ISSKeypoint3D<PointInT, PointOutT, NormalT>::getScatterMatrix (const int& c
   const PointInT& current_point = (*input_)[current_index];
 
   double central_point[3];
-  memset(central_point, 0, sizeof(double) * 3);
+  std::fill_n(central_point, 3, 0);
```

Comment:
Use the declaration statement to initialize to zero (and other places in this file)